src/components/molecules/LoginComponent/LoginSection.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import Header from "../Header";
+import Image from "next/image";
+import BannerImage from "@/../../public/images/banner.png";
+import CopyrightInfo from "@/components/atoms/CopyrightInfo";
+import Link from "next/link";
+import styles from "./LoginSection.module.scss";
+import { MdOutlineAccessTimeFilled } from "react-icons/md";
+import { FaPhoneAlt } from "react-icons/fa";
+
+const LoginSection: React.FC = () => {
+  return (
+    <div className={styles.container}>
+      <Header />
+      <div className={styles.banner}>
+        <Image
+          src={BannerImage}
+          alt="Banner image"
+          layout="fill"
+          objectFit="cover"
+          quality={100}
+        />
+        <div className={styles.overlay} />
+        <h1>My Account</h1>
+      </div>
+      <div className={styles.content}>
+        <div className={styles.forms}>
+          <div className={styles["login-form"]}>
+            <h2>Login</h2>
+            <form>
+              <div className={styles["form-group"]}>
+                <label>Username or email address *</label>
+                <input type="text" placeholder="Enter your email" required />
+              </div>
+              <div className={styles["form-group"]}>
+                <label>Password *</label>
+                <input type="password" placeholder="Enter password" required />
+              </div>
+              <button type="submit">Log in</button>
+              <div className={styles.rememberMe}>
+                <input type="checkbox" id="remember" />
+                <label htmlFor="remember">Remember me</label>
+              </div>
+              <a href="#" className={styles["lost-password"]}>
+                Lost your password?
+              </a>
+            </form>
+            <div className={styles["new-user"]}>
+              New user? <Link href="/signup">sign up here...</Link>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className={styles["branch-section"]}>
```

Comment:
Same issue here. Could you create a Reusable component? Also, review font sizes from our [figma design](https://www.figma.com/design/xiAbsYNAyBYPH5pfPUkIEX/Restaurant-website-template-(Community)?node-id=51-42&node-type=frame&t=tY7j8Bb1oBrfE49Z-0)